cockroachdb/README.md

```diff
@@ -231,9 +231,9 @@ Verify that no pod is deleted and then upgrade as normal. A new StatefulSet will
 
 ### See also
 
-For more information about upgrading a cluster to the latest major release of CockroachDB, see [Upgrade to CockroachDB v20.1](https://www.cockroachlabs.com/docs/stable/upgrade-cockroach-version.html).
+For more information about upgrading a cluster to the latest major release of CockroachDB, see [Upgrade to CockroachDB v20.2](https://www.cockroachlabs.com/docs/stable/upgrade-cockroach-version.html).
 
-Note that there are some backward-incompatible changes to SQL features between versions 19.2 and 20.1. For details, see the [CockroachDB v20.1.0 release notes](https://www.cockroachlabs.com/docs/releases/v20.1.0.html#backward-incompatible-changes).
+Note that there are some backward-incompatible changes to SQL features between versions 20.1 and 20.2. For details, see the [CockroachDB v20.2.0 release notes](https://www.cockroachlabs.com/docs/releases/v20.2.0.html#backward-incompatible-changes).
```

Comment:
Is this statement true? (I expect it is but I haven't checked.)

Reply:
Yeah, the link should work once the release notes are published.